kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -1295,6 +1295,14 @@ object KyuubiConf {
       .timeConf
       .createWithDefault(Duration.ofSeconds(120).toMillis)
 
+  val ENGINE_ALIVE_CLOSE_CONNETION: ConfigEntry[Boolean] =
+    buildConf("kyuubi.session.engine.alive.closeConnectionOnTimeout")
```

Comment:
it's kind of resource leak, I think we should always do close

Reply:
Hi, @pan3793, thank you very much for your comment.
Are you suggesting changing the default value to `true` so that the connection will be always closed?

Reply:
Personally, I would not add configuration but always do close, but if anyone has concerns I think we can add an option set default value to true and mark it as internal. If it works well, we can remove this configuration in the future.

Reply:
+1, we don't need this conf

Reply:
If we rely on probe enabled, we shall turn it on by default too

Reply:
@risyomei  could you remove this config?

Reply:
Sure, I have deleted!